activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+*   Fix `TestFixtures#fixture_path`.
+
+    A previous change deprecated `#fixture_path` in favor of `#fixture_paths`. However the
+    behaviour of `#fixture_path` was also changed to return an array of paths. This update
+    returns it to the original behaviour which is returning a single path. If multiple paths have
+    been configured with `#fixture_paths` the first path will be returned.
+
+    *Andrew Novoselac*
+
```

Comment:
Let's remove this. We only need to have CHANGELOGs for changes on code that was previously released. This isn't the case here.

Reply:
Done